### DISCUSSION TITLE
Can I change the maximum value of the green line for IOB?

### DISCUSSION THREAD

**Discussion by RansomTime** [NightscoutFoundation/xDrip#3051] [Q&A]
Hi, I'm heavily insulin resistant.

When treatments are sent from my pump to xdrip, the green line of treatments often stays at it's maximum value for a while before going down.

![Screenshot_20230826-171417](https://github.com/NightscoutFoundation/xDrip/assets/47819665/3405619c-abf1-4652-b361-cd7cf3b85091)

Is there a setting that allows me to change this maximum value?

**Comment by Navid200** [comment]
@psonnera

**Reply by psonnera** [reply]
@Navid200 I believe it's hard coded. @RansomTime you can see IoB in the status line when it's above max. Changing the scale might require a lot of work...

**Comment by RansomTime** [comment]
Makes sense. Thanks for the reply
